clang/test/CodeGenOpenCL/pipe_builtin.cl

```diff
@@ -1,69 +1,67 @@
-// RUN: %clang_cc1 -triple %itanium_abi_triple -emit-llvm -cl-ext=+cl_khr_subgroups -O0 -cl-std=clc++ -o - %s | FileCheck %s
-// FIXME: Add MS ABI manglings of OpenCL things and remove %itanium_abi_triple
```

Comment:
Test both run lines?

Reply:
Surely I can add the old run line back, but would that bring any value?  Preservation of the pipe access qualifiers is not tested with the old run line for example.

Reply:
It's 2 different paths that both should work, with different output so both should be tested